packages/react-router-permissions/src/authorized-section/authorized-section.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import * as React from 'react';
+import { PermissionsContext } from '../permissions-context/permissions.context';
+
+export type AuthorizedSectionProps = {
+  requires: *,
+  authorizationStrategy?: (*, *) => *,
+  children: ({
+    isAuthorized: boolean,
+  }) => React.Node,
+  [string]: *,
```

Comment:
Is there a way in flow to add them to `children`?

Reply:
I'm afraid I don't get what you mean.